src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,206 @@
+$base-color: #000066;
+$white-bg: #fff;
+$link-color: #939393;
+
+.cardOuter {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  padding: 20px;
+  margin: 200px 30px;
+  justify-content: center;
+  background-color: $white-bg;
+  color: $base-color;
+  position: relative;
+  border-radius: 15px;
+}
+
+.cardMain {
+  flex-basis: 70%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.cardImage {
+  flex-basis: 30%;
+  padding: 0 2em;
+
+  img {
+    height: 150px;
+    border-radius: 10px;
+  }
+}
+
+.cardInfo {
+  flex-basis: 70%;
+}
+
+.cardContact {
+  flex-basis: 30%;
+  display: flex;
+  flex-direction: column;
+  padding-top: 1em;
+}
+
+.cardLinks {
+  flex-basis: 100%;
+}
+
+.contactInfo {
+  flex-basis: 100%;
+}
+
+.textIcon {
+  padding-left: 20px;
+  position: relative;
```

Comment:
again `position:relative`, I can design this layout without using position anywhere in the css.